addon/components/o-s-s/password-input.stories.js

```diff
@@ -23,18 +23,35 @@ export default {
         defaultValue: { summary: false }
       },
       control: { type: 'text' }
+    },
+    validatesFormat: {
+      description: 'Whether or not to validate the password format with the RegEx',
+      table: {
+        type: {
+          summary: 'boolean'
+        },
+        defaultValue: { summary: true }
+      },
+      control: { type: 'boolean' }
+    },
+    validates: {
+      description: 'A callback that indicates wheter or not the current input matches the regex'
     }
   }
 };
 
 const DefaultUsageTemplate = (args) => ({
   template: hbs`
-      <OSS::PasswordInput @value={{this.value}} />
+      <OSS::PasswordInput @value={{this.value}} @validatesFormat={{this.validatesFormat}} @validates={{this.validates}} />
   `,
   context: args
 });
 export const BasicUsage = DefaultUsageTemplate.bind({});
 BasicUsage.args = {
   value: 'myPassword',
-  errorMessage: ''
+  errorMessage: '',
+  validatesFormat: false,
+  validates: (validity) => {
+    console.log('Current input is ' + validity);
+  }
```

Comment:
Maybe use `validates: action('validates')` and import `import { action } from '@storybook/addon-actions';` link in power-select stories. It should automatically display a 'console.log' in action section